### PR TITLE
Implement ICS04 RPCs

### PR DESCRIPTION
**Pull request by junha1**

### CODE REVIEW

File: core/src/ibc/channel_04/manager.rs [hunk 1]
```diff
@@ -15,7 +15,9 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
```
Comment (majecty) [inline core/src/ibc/channel_04/manager.rs]: What do you mean by the commit message "Refactor ICS data that are stored in hash"?
Is "in hash" commonly used term?

Comment (junha1) [reply core/src/ibc/channel_04/manager.rs]: `in hash` was meant to be `as a hash`. @ScarletBlue answered that `as a hash` would be more natural. I'll update it

**Comment by majecty** [comment]
@junha1 Could you fix the conflict?

**Comment by junha1** [comment]
@majecty done

**Pull request merged**